<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1168</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1168</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuestionnaire"/>
   <addaction name="separator"/>
   <addaction name="actionEditQuestions"/>
   <addaction name="separator"/>
   <addaction name="actionReportMain"/>
   <addaction name="separator"/>
   <addaction name="actionPlace"/>
   <addaction name="actionRegion"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <action name="actionQuestionnaire">
   <property name="text">
    <string>Анкеты</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;анкеты&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionEditQuestions">
   <property name="text">
    <string>Редактор вопросов</string>
   </property>
   <property name="toolTip">
    <string>Редактирование вопросов</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionRegion">
   <property name="text">
    <string>Районы</string>
   </property>
   <property name="toolTip">
    <string>Районы</string>
   </property>
  </action>
  <action name="actionPlace">
   <property name="text">
    <string>Места проведения</string>
   </property>
   <property name="toolTip">
    <string>Места опроса (МО)</string>
   </property>
  </action>
  <action name="actionReportMain">
   <property name="text">
    <string>Отчет</string>
   </property>
   <property name="toolTip">
    <string>Отчет по анкетам</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
